_includes/faq.html

```diff
@@ -0,0 +1,34 @@
+
+			<p class="a">We also strongly encourage members of groups that are underrepresented in science to apply. Diversity is one of our explicit evaluation criteria, and we work hard to try and make Neurohackademy a comfortable and welcoming environment for people of all races, genders, orientations, nationalities, and other backgrounds.</p>
+
+			<h5>I'm an early-career researcher with a strong background in a computational field, but little or no experience in neuroscience. Can I attend?</h5>
+
+			<p class="a">While we don't expect most applicants to have a very strong background in neuroscience, we do impose a soft requirement that applicants should have at least <i>some</i> prior experience working with neuroscience data. It's important for us to make sure that participants with computational backgrounds are confident they'll enjoy working on neuroscience problems during (and ideally beyond) the course, and have sufficient familiarity with basic neuroscience concepts to benefit from the materials and projects. So if you have a background in, say, physics or computer science, but have never touched a neuroscience dataset, this probably isn't the right course for you. If, on the other hand, you've had some prior experience (possibly in a collaborative context) working with neuroscience data (e.g., you've helped write algorithms to process or analyze brain data), and enjoyed it, you should absolutely apply.</p>
+
+			<h5>I'm an early-career researcher with a strong background in neuroscience, but my computational skills aren't very strong. Can I attend?</h5>
+
+			<p class="a">The primary goal of Neurohackademy is to help researchers develop the computational skills they need in order to make the most of large, rich neuroscience datasets. So we certainly don't expect applicants to come in with extensive prior programming experience (and if anything, that might hurt one's chances of admission, since there's no point in us trying to teaching someone things they already know!). That said, we do assume that all participants have a minimal level of programming competency--without which a participant will probably find it much more difficult to benefit from the course.</p>
```

Comment:
I'd remove the parenthetical "(and if anything..." -- it just opens up more questions, in my opinion